sessionCheckGlobal.js

```diff
@@ -92,6 +95,36 @@
         idTokenRequestCooldown.call(this);
     };
 
+    /**
+     * Destroys the OIDC session check instance to allow garbage collection; removes the iframe and associated iframe event listeners.
+     * Recommend dereferencing of session check after destruction to prevent use of impotent SessionCheck instance.
+     * @example
+     * // Good example
+     * this.sc = new SessionCheck(config)
+     * // use this.sc, then...
+     * this.sc.destroy()
+     * this.sc = null
+     * @example
+     * // Good example
+     * function() {
+     *   const sc = new SessionCheck(config)
+     *   // use sc, then...
+     *   sc.destroy()
+     * } // at the end of the function the session check goes out of scope and is cleaned up by the garbage collector, along with the iframe and event handler.
+     * @example
+     * // Bad example
+     * this.sc = new SessionCheck(config)
+     * // use this.sc, then...
+     * this.sc = null // does not remove iframe or event listener, meaning events will still occur and the garbage collector will not collect this.sc
+     */
```

Comment:
Could you please also update README.md with the examples / explanation for the use of this function? Thanks

Reply:
I added a little section to the bottom of "Using this library". Please let me know if you have any more specific doc-related suggestions.